src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -28,5 +31,6 @@
       <i aria-hidden="true" class="fa fa-clock-o"></i>
       <pretty-duration from="timefilter.time.from" to="timefilter.time.to"></pretty-duration>
     </button>
+    <i ng-click="foward()" class="time-nav-btn fa fa-arrow-right ng-scope"></i>
   </li>
 </ul>
```

Comment:
I think this looks good. I'm not _100%_ confident about it but that's because I haven't used it as much as you have. Once I've had the chance to use it for awhile I'm sure I'll either be on board or be able to provide some better suggestions. :)